app/lib/dns.server.ts

```diff
@@ -21,14 +23,34 @@ if (process.env.NODE_ENV === 'production') {
   }
 }
 
+const { AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY } = secrets;
+
+const credentials = () => {
+  if (process.env.NODE_ENV === 'production') {
+    if (!AWS_ACCESS_KEY_ID) {
+      throw new Error('Missing AWS_ACCESS_KEY_ID secret');
+    }
+    if (!AWS_SECRET_ACCESS_KEY) {
+      throw new Error('Missing AWS_SECRET_ACCESS_KEY secret');
+    }
+
+    return {
+      accessKeyId: AWS_ACCESS_KEY_ID,
+      secretAccessKey: AWS_SECRET_ACCESS_KEY,
+    };
+  }
+
+  return {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID || AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || AWS_SECRET_ACCESS_KEY,
+    sessionToken: process.env.AWS_SESSION_TOKEN,
```

Comment:
For my better understanding, these credentials are used in `staging` as we don't have any credentials for DNS in `dev` environment.

Reply:
This gives the ability to provide them if you want (e.g., if you want to connect to real AWS locally in dev).